website/docs/openshift/microshift/index.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 104
+title: MicroShift
+description: Create a MicroShift cluster with Podman Desktop
+keywords: [podman desktop, podman, minc extension, microshift]
+tags: [install-minc-extension, create-a-microshift-cluster]
+---
+
+# Creating a MicroShift cluster
+
+With Podman Desktop, you can use the MicroShift in a container (MINC) extension to create a lightweight Kubernetes cluster that runs on minimal resources, such as CPU, RAM, and storage. The cluster is deployed as a container within your Podman environment. After setting up a MicroShift cluster, you can:
+
+- Develop Kubernetes applications in a resource-constrained environment.
+- Provision lightweight Kubernetes control planes.
+- Run your Kubernetes workloads at the edge or even in network-constrained situations, such as low connectivity or node access issues.
+- Access the images built with Podman from your MicroShift cluster.
+- Have a consistent development and management experience with an OpenShift Container Platform (OCP) cluster.
+
+#### Prerequisites
+
+- A running [Podman machine](/docs/podman/creating-a-podman-machine) with root privileges.
+- [Install the MINC extension](/docs/extensions/install).
+- On Windows: [Enable the `cgroup v2` kernel feature](https://stackoverflow.com/questions/73021599/how-to-enable-cgroup-v2-in-wsl2) in the Windows Subsystem for Linux (WSL).
```

Comment:
I could not find any official documentation for this part. If we can link to any other specific document, please point to it.

Reply:
https://github.com/minc-org/minc?tab=readme-ov-file#windows

Reply:
I think we should add a page for it